templates/server-configmap.yaml

```diff
@@ -105,8 +105,13 @@ data:
           tags:
             type: frontend
           prometheus:
+            framework: "opentelemetry"
             timerType: {{ default .Values.server.metrics.prometheus.timerType .Values.server.frontend.metrics.prometheus.timerType }}
             listenAddress: "0.0.0.0:9090"
+          prometheusSDK:
```

Comment:
what does promtheus SDK mean? - like why is it called that. and why do we need both? do we need to scrape both 9090 and 9091?

Reply:
We need both. We have to report metrics for SDK via tally due to sdk not using opentelemetry yet. OT and SDK can't serve from same address unfortunately.